### ISSUE TITLE
Conversion from Swagger spec 1.2 to 2.0 adds HTTP 200 response if not present

### ISSUE THREAD

**Issue by robertmorrell-inttra** [LucyBot-Inc/api-spec-converter#136]
I didn't see this issue brought up in the other issues, but I am experiencing a problem when i define an API that does not use an HTTP 200 (OK) response, (instead it used a 204), and during conversion a 200 response is added to the output json which an invalid reference.

Example: 1.2 spec:
"responseMessages" : [ {
        "code" : 204,
        "message" : "No-Content - Updated successfully"
      }, {
        "code" : 400,
        "message" : "Invalid parameters passed",
        "responseModel" : "ExceptionModel"
      }, {
        "code" : 500,
        "message" : "An error has occurred",
        "responseModel" : "ExceptionModel"
      } ]

Converted 2.0 spec
"responses": {
          "200": {
            "description": "No response was specified",
            "schema": {
              "$ref": "#/definitions/Response"
            }
          },
          "204": {
            "description": "No-Content - Updated successfully"
          },
          "400": {
            "description": "Invalid parameters passed",
            "schema": {
              "$ref": "#/definitions/ExceptionModel"
            }
          },
          "500": {
            "description": "An internal error has occurred",
            "schema": {
              "$ref": "#/definitions/ExceptionModel"
            }
          }
        }

**Comment by MikeRalphson** [comment]
I have raised an issue as above with the `swagger-converter` project which is used to convert Swagger 1.2 definitions to Swagger/OpenAPI 2.0.

**Comment by robertmorrell-inttra** [comment]
Ok thank you. Does that help this project as well?

**Comment by MikeRalphson** [comment]
@robertmorrell-inttra yes, that module is used by `api-spec-converter` for Swagger 1.2 definitions.

**Comment by robertmorrell-inttra** [comment]
I see. Thank you.

**Comment by bobby-brennan** [comment]
For reference, here's the issue this depends on: https://github.com/apigee-127/swagger-converter/issues/78

Once that's closed we'll update the dependency here.

**Comment by joepeding** [comment]
@rbren This issue has been solved in swagger-converter version 2.0.0.
